bucket/lamexp.json

```
@@ -0,0 +1,40 @@
+{
+    "homepage": "http://lamexp.sourceforge.net",
+    "version": "4.17.2188",
+    "description": "Audio Encoder Front-End.",
+    "license": "GPL-2.0-or-later",
+    "url": "https://downloads.sourceforge.net/project/lamexp/LameXP.2018-11-10.Release-Static.Build-2188.zip",
+    "hash": "sha1:a54dccfe83fdd13d97ab3d80be1d81402a95ea4e",
+    "depends": "nero-aac",
+    "pre_install": [
+        "Rename-Item \"$dir\\LameXP.exe\" \"$dir\\LameXP-Portable.exe\"",
+        "$ini = \"LameXP-Portable.ini\"",
+        "If (!(Test-Path \"$persist_dir\\$ini\")) {",
+        "   New-Item \"$dir\\$ini\" | Out-Null",
+        "} Else {",
+        "   $version -match \"(\\d+).(\\d+).(\\d+)\" | Out-Null",
+        "   $cleanVersion = $Matches[1] + $Matches[2] + '0' + $Matches[3]",
+        "   (Get-Content \"$persist_dir\\$ini\") | Foreach-Object { $_ -replace \"LameXP_\\d+\", \"LameXP_$cleanVersion\" } | Set-Content \"$persist_dir\\$ini\"",
+        "},",
+        "$aac = versiondir 'nero-aac' 'current'",
+        "If (Test-Path $aac) {",
+        "   info \"Adding Nero AAC Codec Support\"",
+        "   Copy-Item \"$aac\\win32\\*.exe\" $dir",
```

Not quoeted dir variable
```suggestion
        "   Copy-Item \"$aac\\win32\\*.exe\" \"$dir\"",
```